src/components/NewWorkspaceModal.js

```diff
@@ -132,6 +138,7 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
   // Lifecycle
   useOnMount(() => {
     loadData()
+    loadAlphaRegionalityUser()
```

Comment:
Does this work as intended? Asking since `loadAlphaRegionalityUser` doesn't seem to be a function? Could we inline like below?
```suggestion
    reportErrorAndRethrow('Error loading user group membership')(async () => {
    setIsAlphaRegionalityUser(await Ajax(signal).Groups.group(getConfig().alphaRegionalityGroup).isMember())
  })
```

Reply:
loadAlphaRegionalityUser is defined on lines 74-77. Am I missing something?

Reply:
Ah never mind...I'd misread the definition of `reportErrorAndRethrow` which is called by `loadAlphaRegionalityUser`.